SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerMultiPossessionTest.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "CrossServerMultiPossessionTest.h"
+
+#include "Containers/Array.h"
+#include "EngineClasses/SpatialNetDriver.h"
+#include "GameFramework/GameModeBase.h"
+#include "GameFramework/PlayerController.h"
+#include "GameMapsSettings.h"
+#include "LoadBalancing/LayeredLBStrategy.h"
+#include "SpatialFunctionalTestFlowController.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/TestMovementCharacter.h"
+#include "TestPossessionPawn.h"
+#include "TestPossessionPlayerController.h"
+#include "Utils/SpatialStatics.h"
+
+/**
```

Comment:
The test description is a bit confusing.
It mentions two client workers, then 3 "Num Required Clients"
It also mentions steps to take to make the test work, like selecting the game mode and multi worker settings.
Mentioning them is alright, like "this test expects a 2x2 load balancing grid and ACrossServerPossessionGameMode", but in practice, these tests should have accompanying maps in the GDK Gyms project which already provides the correct setup (we have future plans to automate this, but that's a matter for another time) .

More details about the actual scenario being conducted would be appreciated.

Reply:
updated, Please take a look, thanks

Reply:
Once again, the setup section is rather useless for what should be an automated test.
What is missing is the description of the test initial state, what operation are done, and what is the expected result (under Test:)
The format that was in CrossServerPossessionLockTest, with Setup actually describing what and where the actor were was much more useful (even though the description itself was incorrect).